### DISCUSSION TITLE
Nuget for ACadSharp

### DISCUSSION THREAD

**Discussion by Spencer-reborn-Elliott** [DomCR/ACadSharp#258] [Show and tell]
I use visual studio as my main IDE and often use the Nuget Package Manager. Sadly, despite the ACadSharp library being available via Nuget, when I searched for it in the Browse facility with the package source being nuget.org it doesn't show up. 

The only package that does show is "AcadLib" - Which does relate to CAD, but is something entirely different. I am sure people have accidentally installed this thinking it is ACadSharp. 

The only way I could install via Nuget is on the Nuget Package Manager Console, using the following command:

> NuGet\Install-Package ACadSharp -Version 2.0.3-beta

For anyone not familiar with how to get the Nuget console, in Visual Studio simply go to "Tools", then "Nuget Package Manager" and then "Package Manager Console".  

It's fine for me - Just thought I would post this here incase anyone was struggling to get the package (obviously the version will change, so anyone reading this in the future will probably want to check what the latest version is an update the command to correspond).

**Comment by DomCR** [comment]
Hi @Spencer-reborn-Elliott 

The package is still in a beta version, so is listed as a prerelease version, if you want to be able to see the package in the Nuget browser you have to check the "Include prerelease"

![image](https://github.com/DomCR/ACadSharp/assets/46492487/41203369-8b13-42a5-9f8c-b87f330b7c9f)

That way you can see it.
